index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav id="navbar">
+      <a href="#About" class="nav-links">About</a>
+      <a href="#projects" class="nav-links">Projects</a>
+      <a href="#contact" class="nav-links">Contact</a>
+    </nav>
+<main>
+    <div id="About">
+      <div id="welcome-section">
+        <h1>Hey I am a Web Developer</h1>
+        <p>To check out my project Scroll-Down</p>
+      </div>
+    </div>
+    <div id="projects">
+        <h2>These are some of my projects</h2>
+      <div class="project-tile">
+      <img src="project.png" alt="">
+      <img src="clone.png" alt="">
```

Comment:
Please always try to add the alt tag text in your img tag. You can imagine a scenario where a user's internet is very slow and not able to load the media of the website, then in that kind of situation, this alt tag will help the user to understand what the media is all about.